pkg/event_processor/bash_parser.go

```diff
@@ -0,0 +1,62 @@
+package event_processor
+
+import (
+	"context"
+	"ecapture/user/config"
+	"errors"
+
+	"golang.org/x/sys/unix"
+)
+
+type BashParser struct {
+	line   string
+	isdone bool
+}
+
+func (b *BashParser) Display() []byte {
+	return []byte("Line: " + b.line)
+}
+
+func (b *BashParser) Init() {
+	b.isdone = false
+	b.line = ""
+}
+
+func (b *BashParser) IsDone() bool {
+	return b.isdone
+}
+
+func (b *BashParser) Name() string {
+	return "BashParser"
+}
+
+func (b *BashParser) PacketType() PacketType {
+	return PacketTypeBash
+}
+
+func (b *BashParser) ParserType() ParserType {
+	return ParserTypeBash
+}
+
+func (b *BashParser) Reset() {
+	b.isdone = false
+	b.line = ""
+}
+
+func (bp *BashParser) Write(b []byte) (int, error) {
+	bp.line += unix.ByteSliceToString((b[:]))
+	return len(bp.line), nil
+}
+
+func (bp *BashParser) detect(ctx context.Context, b []byte) error {
+	if ctx.Value(config.CONTEXT_KEY_MODULE_NAME) == "EBPFProbeBash" {
```

Comment:
I think the design here is not elegant. The pkg directory should be a business-independent third-party package, and the config package should not be referenced.

Reply:
1. In my implementation, it's necessary to determine whether to use the bash parser based on context information.
2. When setting a key in the context, it cannot be a string or any other built-in type (see https://staticcheck.io/docs/checks#SA1029).
3. Therefore, a custom type contextKey is required, which needs to be introduced in "ecapture/pkg/event_processor" and "ecapture/cli/cmd" as the type for context information keys.
4. I didn't delve too deeply into where to define this type in the config package. Where would be the most appropriate place to put it?"

Reply:
Indeed, abstraction here is a bit troublesome. I'll think about it later too